src/asmcnc/keyboard/custom_keyboard.py

```diff
@@ -154,13 +138,33 @@ def set_keyboard_background(self):
             self.margin_hint = [.05, .06, .05, .06]  # Default margin
             self.background = "atlas://data/images/defaulttheme/vkeyboard_background"
 
+        if self.layout == self.numeric_layout:
+            self.width = Window.width / 3
+        else:
+            self.width = Window.width
+
```

Comment:
This should use scaling_utils.Width instead of Window.width

Reply:
probably need to merge master to get it

Reply:
This works fine but I don't really see the point of it...

Reply:
This will not scale on my laptop. Using scaling utils module it will.

Reply:
For context, on my Mac, Window.size doesn't stay at 800x480 or 1280x768 even if you set it. So using scaling_utils I can enforce it.